README.md

```diff
@@ -1,24 +1,585 @@
+
+[gem-simplecov]: https://img.shields.io/badge/simplecov-0.22.0-yellow?style=flat-square
+[gem-simplecov-url]: https://github.com/simplecov-ruby/simplecov
+
+[gem-factory_bot_rails]: https://img.shields.io/badge/factory_bot_rails-6.4.0-success?style=flat-square
+[gem-factory_bot_rails-url]: https://github.com/thoughtbot/factory_bot_rails
+
+[gem-faker]: https://img.shields.io/badge/faker-3.2.2-red?style=flat-square
+[gem-faker-url]: https://github.com/faker-ruby/faker
+
+[gem-pry]: https://img.shields.io/badge/pry-0.14.2-yellow?style=flat-square
+[gem-pry-url]: https://github.com/pry/pry
+
+[gem-shoulda-matchers]: https://img.shields.io/badge/shoulda--matchers-6.0.0-orange?style=flat-square
+[gem-shoulda-matchers-url]: https://github.com/thoughtbot/shoulda-matchers
+
+[gem-faraday]: https://img.shields.io/badge/faraday-2.8.1-yellowgreen?style=flat-square
+[gem-faraday-url]: https://github.com/lostisland/faraday
+
+[gem-figaro]: https://img.shields.io/badge/figaro-1.2.0-critical?style=flat-square
+[gem-figaro-url]: https://github.com/laserlemon/figaro
+
+[gem-jsonapi-serializer]: https://img.shields.io/badge/jsonapi--serializer-2.2.0-blue?style=flat-square
+[gem-jsonapi-serializer-url]: https://github.com/jsonapi-serializer/jsonapi-serializer
+
+[gem-capybara]: https://img.shields.io/badge/capybara-3.39.2-brightgreen?style=flat-square
+[gem-capybara-url]: https://github.com/teamcapybara/capybara
+
+[gem-webmock]: https://img.shields.io/badge/webmock-3.19.1-yellowgreen?style=flat-square
+[gem-webmock-url]: https://github.com/bblimke/webmock
+
+[gem-vcr]: https://img.shields.io/badge/vcr-6.2.0-orange?style=flat-square
+[gem-vcr-url]: https://github.com/vcr/vcr
+
+
+<!-- CONTRIBUTOR SHIELDS AND URLS -->
+[github-shield-zh]: https://img.shields.io/badge/GitHub-XanderHendry-success?style=for-the-badge&logo=github
+[github-url-zh]: https://github.com/xanderhendry
+
+[github-shield-st]: https://img.shields.io/badge/GitHub-Sykogst-success?style=for-the-badge&logo=github
+[github-url-st]: https://github.com/Sykogst
+
+[github-shield-kz]: https://img.shields.io/badge/GitHub-zkevkev-success?style=for-the-badge&logo=github
+[github-url-kz]: https://github.com/zkevkev
+
+[github-shield-eh]: https://img.shields.io/badge/GitHub-ericahagle-success?style=for-the-badge&logo=github
+[github-url-eh]: https://github.com/ericahagle
+
+[github-shield-ar]: https://img.shields.io/badge/GitHub-tenthwalker-success?style=for-the-badge&logo=github
+[github-url-ar]: https://github.com/tenthwalker
+
+<!-- LINKEDIN SHIELDS AND URLS-->
+[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=for-the-badge&logo=linkedin&colorB=555
+[linkedin-url-st]: https://www.linkedin.com/in/sam-t-tran
+[linkedin-url-kz]: https://www.linkedin.com/in/kevin-zolman
+[linkedin-url-zh]: https://www.linkedin.com/in/xander-hendry-70b804289
+[linkedin-url-eh]: https://www.linkedin.com/in/ericahagle
+[linkedin-url-ar]: https://www.linkedin.com/in/arden-ranta/
```

Comment:
I absolutely adore the tags for contributors. I've never thought to do that, but am totally going to for the fe repo!